lib/passport-wsfed-saml2/saml.js

```diff
@@ -322,14 +337,11 @@ SAML.prototype.parseAssertion = function(samlAssertion, callback) {
   if (!samlAssertion.getAttribute)
     samlAssertion = samlAssertion.documentElement;
 
-  var version;
-  if (samlAssertion.getAttribute('MajorVersion') === '1')
-    version = '1.1';
-  else if (samlAssertion.getAttribute('Version') === '2.0')
-    version = '2.0';
-  else
+  const version = utils.getSamlAssertionVersion(samlAssertion);
+  const supportedVersions = ['1.1', '2.0'];
+  if (supportedVersions.indexOf(version) === -1){
```

Comment:
Since you have the method above checking if the version is known, can't you just check `if (version)` and remove the extra inclusion check?

Reply:
The thinking was that we might update the util function with new versions, in whcih case `supportedVersions` doesn't carry any meaning. For instance, 6 months from now we return `2.2` from  `utils.getSamlAssertionVersion`


The old version had
```

  var version;
  if (samlAssertion.getAttribute('MajorVersion') === '1')
    version = '1.1';
  else if (samlAssertion.getAttribute('Version') === '2.0')
    version = '2.0';
  else
    return callback(new Error('SAML Assertion version not supported'), null);

```
and my thinking was to separate the logic that gets the version from the decision whether the version was supported. But I can change it to just check if the `version` is defined and remove the list of supported versions, of course.